js/apps/admin-ui/src/user-federation/ldap/LdapSettingsAdvanced.tsx

```diff
@@ -127,7 +154,7 @@ export const LdapSettingsAdvanced = ({
           label={t("trustEmail")}
           labelIcon={
             <HelpItem
-              helpText={t("user-federation-help:trustEmailHelp")}
+              helpText="user-federation-help:trustEmailHelp"
```

Comment:
don't change this

Reply:
Sorry, I am a bit confused - which should stay? I am not familiar with this review process, so I don't know what I should do, @edewit

Reply:
it should be `helpText={t("user-federation-help:trustEmailHelp")}` you didn't need to change that line. The only thing that is needed is the added Field further up in the same file